### PR TITLE
Add variant right hand panel overview content

### PR DESCRIPTION
**Pull request by jyothishnt**
## Description
Add overview content on variant entity viewer side bar

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-2379

## Deployment URL(s)
http://variant-sidebar.review.ensembl.org

## Views affected
Variant entity viewer

### CODE REVIEW

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/VariantViewSideBar.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { useAppSelector } from 'src/store';
+
+import Sidebar from 'src/shared/components/layout/sidebar/Sidebar';
+
+import { getEntityViewerSidebarTabName } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/VariantViewSideBar.tsx:23]: Should we even bother with sidebar tabs at this stage? I remember Andrea said that she wasn't sure she will have use for the second tab (the one that says alternative IDs and locations in XD).

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+  return (
+    <div className={styles.accordionContainer}>
+      <Accordion
+        className={styles.entityViewerAccordion}
+        allowMultipleExpanded={true}
+      >
+        <AccordionItem
+          className={styles.entityViewerAccordionItem}
+          uuid={'function'}
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx:52]: could you start this accordion as open? It is closed now.

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+        <div className={styles.label}>Ref</div>
+        <div className={styles.value}>
+          {referenceAllele?.reference_sequence}
+        </div>
+      </div>
+      <div className={styles.row}>
+        <div className={styles.label}>Alt</div>
+        <div className={styles.value}>
+          {alternativeAlleles.map((allele, index) => (
+            <div key={index}>{allele.allele_sequence}</div>
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx:134]: Could you ask Andrea how to deal with long alts?

For example, try `variant:21:45988497:rs4647797`.

_(Not a blocker for this PR; can be addressed later)_

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    return null;
+  }
+
+  const currentVariantGroup = variantGroups.find(({ variant_types }) =>
+    variant_types.find(({ label }) => label === variantConsequence)
+  );
+
+  const currentVariantType = currentVariantGroup?.variant_types.find(
+    ({ label }) => label === variantConsequence
+  );
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx:72]: The label for the group is plural (e.g. "Protein altering variants", "Transcript variants"). We will want it in a singular form.

_(Not a blocker for this PR)_

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+            ) : (
+              <span>{currentVariantType.so_accession_id}</span>
+            ))}
+        </div>
+        <div className={styles.newRowGroup}>
+          <span className={styles.labelText}>{currentVariantGroup?.label}</span>
+        </div>
+      </section>
+
+      <section></section>
```
Comment (azangru) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx:113]: this line, and the commented code below — ✂️

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/VariantViewSideBar.tsx (hunk 1 above)
Comment (jyothishnt) [reply src/content/app/entity-viewer/variant-view/variant-view-sidebar/VariantViewSideBar.tsx:23]: I think we should leave it. Andrea said there will be a second tab. She also said to create a dummy tab. I will see if I can add it to this PR. If not, its ok not a blocker

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx (hunk 3 above)
Comment (jyothishnt) [reply src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx:134]: Added 18ch but it doesn't really take exact char length. How to solve this?

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+import {
+  Accordion,
+  AccordionItem,
+  AccordionItemHeading,
+  AccordionItemPanel,
+  AccordionItemButton
+} from 'src/shared/components/accordion';
+
+import { getReferenceAndAltAlleles } from 'src/shared/helpers/variantHelpers';
+import { EntityViewerVariantDefaultQueryResult } from 'src/content/app/entity-viewer/state/api/queries/variantDefaultQuery';
```
Comment (imransl) [inline src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx:29]: Type import?

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx (hunk 3 above)
Comment (azangru) [reply src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/MainAccordion.tsx]: Change the string in javascript :-)

File: src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx (hunk 4 above)
Comment (jyothishnt) [reply src/content/app/entity-viewer/variant-view/variant-view-sidebar/overview/VariantOverview.tsx:72]: Done, updated variant types file.

**Review by azangru — APPROVED** [review]
Ok, let's merge it.

**Pull request merged**